huxley/utils/test.py

```diff
@@ -96,11 +96,38 @@ def new_committee(**kwargs):
 class TestCountries():
     @staticmethod
     def new_country(**kwargs):
-        c = Country(name=kwargs.pop('name', 'TestCountry'),
-                    special=kwargs.pop('special', False))
+        c = Country(
+                name=kwargs.pop('name', 'TestCountry'),
+                special=kwargs.pop('special', False))
+        c.save()
+        return c
+
+class TestDelegates():
+    @staticmethod
+    def new_delegate(**kwargs):
+        a = TestAssignments.new_assignment()
+        c = Delegate(
```

Comment:
Might make more sense to assign this to a variable called `d`. Just for consistency